index.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Ayoub Chahir | Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li id="logo"><a href="#" class="logo">My Logo</a></li>
+          <li>
+            <img src="./img/sandwich.svg" alt="" />
+          </li>
+        </ul>
+      </nav>
+      <div class="header-background-shapes">
+        <img src="./img/Ellipse-3.svg" class="ellipse-3" alt="" /><img
+          src="./img/Ellipse-4.svg"
+          class="ellipse-4"
+          alt=""
+        /><img src="./img/Ellipse-5.svg" class="ellipse-5" alt="" /><img
+          src="./img/Ellipse-8.svg"
+          class="ellipse-8"
+          alt=""
+        /><img src="./img/Ellipse-11.svg" class="ellipse-11" alt="" /><img
+          src="./img/Ellipse-12.svg"
+          class="ellipse-12"
+          alt=""
+        /><img src="./img/Ellipse-13.svg" class="ellipse-13" alt="" /><img
+          src="./img/Ellipse-14.svg"
+          class="ellipse-14"
+          alt=""
+        /><img src="./img/Ellipse-15.svg" class="ellipse-15" alt="" /><img
+          src="./img/Rectangle-4.svg"
+          class="rect-4"
+          alt=""
+        /><img src="./img/Rectangle-5.svg" class="rect-5" alt="" /><img
+          src="./img/Rectangle-6.svg"
+          class="rect-6"
+          alt=""
+        /><img src="./img/Rectangle-7.svg" class="rect-7" alt="" />
+      </div>
+      <section class="headline">
+        <h1>
+          I'm Ayoub <br />
+          Glad to see you!
+        </h1>
+        <p class="presentation">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+        <p class="connect">Let's connect</p>
+        <ul class="social-media">
+          <li class="flex-item">
+            <a href=""><img src="./img/twitter.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/Linkedin-icon.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/medium.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/github.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""
+              ><img src="./img/angellist.svg" alt="" class="flex-item"
+            /></a>
+          </li>
+        </ul>
+      </section>
+    </header>
+    <div class="works">
```

Comment:
Kindly use a `section` tag rather than a `div` to meet the project requirements 📝 All the works section should be wrapped with this particular element

Reply:
I will have a linter error, because sections need to have headings, and I don't need one, so I didn't used section

Reply:
- [ ] Please check this [link ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)to read more about `section tag` 
- [ ] Suggestion: try to use this following structure for the works section:
```html
 <section class="card-works">
    <h2 class="project-title">Tonic</h2>
   ...
   <div class="card-works-1"></div>
   <div class="card-works-1"></div>
   <div class="card-works-1"></div>
   ...
</section>
```